tests/phpunit/integration/Core/Util/Debug_DataTest.php

```diff
@@ -55,7 +55,6 @@ public function test_register() {
 				'verification_status',
 				'connected_user_count',
 				'active_modules',
-				'recoverable_modules',
```

Comment:
It could be worth adding an additional test case here to cover the Dashboard Sharing fields...